tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java

```diff
@@ -117,7 +122,7 @@ public void journalSync() throws Exception {
     checkDb(tableMasterRestart, DB_NAME, oldInfo);
     tableMasterRestart.syncDatabase(DB_NAME);
```

Comment:
This line resyncs the database.. that's why the test is passing without your change. @hycsam